src/contractor/contractor.cpp

```diff
@@ -95,10 +98,11 @@ EdgeWeight getNewWeight(IterType new_speed_iter, const double segment_length, Ed
     {
         auto newSecs = new_segment_weight / 10;
         auto oldSecs = oldWeight / 10;
```

Comment:
the types here are ints, right. so /10 might result in a loss of precision. I'd prefer /10.0 and a formatted output to not have hundreds of decimals in there.